docs/asyncapi.yaml

```diff
@@ -0,0 +1,158 @@
+asyncapi: 2.4.0
+info:
+  title: FFC Payment Statement Data
+  version: '1.0.0'
+  description: Publish datasets from Data Warehouse for Payment Statements
+channels:
+  organisation:
+    publish:
+      message:
+        $ref: '#/components/messages/Organisation'
+  calculation:
+    publish:
+      message:
+        $ref: '#/components/messages/Calculation'
+components:
+  messages:
+    Organisation:
+      name: Organisation
+      contentType: application/json
+      payload:
+        $ref: '#/components/schemas/Organisation'
+    Calculation:
+      name: Calculation
+      contentType: application/json
+      payload:
+        $ref: '#/components/schemas/Calculation'
+  schemas:
+    Organisation:
+      type: object
+      properties:
+        name:
+          $ref: '#/components/schemas/BusinessName'
+        sbi:
+          $ref: '#/components/schemas/SBI'
+        frn:
+          $ref: '#/components/schemas/FRN'
+        addressLine1:
+          $ref: '#/components/schemas/AddressLine1'
+        addressLine2:
+          $ref: '#/components/schemas/AddressLine2'
+        addressLine3:
+          $ref: '#/components/schemas/AddressLine3'
+        city:
+          $ref: '#/components/schemas/City'
+        county:
+          $ref: '#/components/schemas/County'
+        postcode:
+          $ref: '#/components/schemas/Postcode'
+        emailAddress:
+          $ref: '#/components/schemas/Email'
+        updated:
+          $ref: '#/components/schemas/Updated'
+    Calculation:
+      type: object
+      properties:
+        calculationId:
+          $ref: '#/components/schemas/CalculationId'
+        sbi:
+          $ref: '#/components/schemas/SBI'
+        frn:
+          $ref: '#/components/schemas/FRN'
+        calculationDate:
+          $ref: '#/components/schemas/CalculationDate'
+        invoiceNumber:
+          $ref: '#/components/schemas/InvoiceNumber'
+        scheme:
```

Comment:
I assume this is useful to allow us to differentiate between land based schemes and non-land based schemes so we know how to properly save the record down?

Reply:
Calculation date is a date item in part 1 of the current statement and is on the BPS claim statement so we've asked for it for DWH.  I actually think with the intent of that part, settlement date would be more appropriate but still being discussed (I believe)

The invoice number is the original Siti Agri invoice number before we convert it so we'll need it to relate the calculation to the payment request.

As with every property on these payments, I'm happy to rename them to anything and any format that makes life easier for saving them in `ffc-pay-statement-constructor`.